services/apigateway/src/it/java/software/amazon/awssdk/services/apigateway/GetExportIntegrationTest.java

```diff
@@ -1,74 +0,0 @@
-/*
- * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License").
- * You may not use this file except in compliance with the License.
- * A copy of the License is located at
- *
- *  http://aws.amazon.com/apache2.0
- *
- * or in the "license" file accompanying this file. This file is distributed
- * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- * express or implied. See the License for the specific language governing
- * permissions and limitations under the License.
- */
-
-package software.amazon.awssdk.services.apigateway;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-import java.io.IOException;
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
-import org.junit.Ignore;
-import org.junit.Test;
-import software.amazon.awssdk.core.SdkBytes;
-import software.amazon.awssdk.testutils.Waiter;
-
-/**
- * Integration tests for {@link software.amazon.awssdk.services.apigateway.model.GetExportRequest}.
- */
-@Ignore // Running during release causing TooManyRequests exception
-public class GetExportIntegrationTest extends IntegrationTestBase {
```

Comment:
I think we might want to keep this one because this one verified a bug fixed by @dagnir 

Just need to figure out how we can get rid of `@ignore`

Reply:
I deleted one of the two APIG tests not realizing this was the one that tested the interceptor so will def add it back in.